app/repositories/SessionCacheRepository.scala

```diff
@@ -106,28 +109,28 @@ case class AccountLinksMongo(accountLinks: Seq[AccountLink], lastUpdated: LocalD
 
 object AccountLinksMongo {
 
-  implicit lazy val writes: OWrites[AccountLinksMongo] = {
+  implicit lazy val writes: OWrites[AccountLinksMongo] =
     (
       (__ \ "accountLinks").write[Seq[AccountLink]] and
         (__ \ "lastUpdated").write(MongoJavatimeFormats.localDateTimeWrites)
-      )(accLinks => Tuple.fromProductTyped(accLinks))
-  }
-  implicit lazy val reads: Reads[AccountLinksMongo] = {
+    )(accLinks => Tuple.fromProductTyped(accLinks))
+  implicit lazy val reads: Reads[AccountLinksMongo]    =
```

Comment:
it might be worth putting a new line between this implicit and the line above for readability. also is it the compiler warning making the equals go out so far to the right?